### PR TITLE
support Atsmart Z6

### PR DESCRIPTION
**Pull request by manhere**
support Atsmart Z6 -- 3 gang smart wall switch (no neutral wire)

### CODE REVIEW

File: devices.js [hunk 1]
```diff
@@ -15490,6 +15490,29 @@ const devices = [
         extend: generic.light_onoff_brightness_colortemp,
     },
 
+    // Atsmart
+    {
+        zigbeeModel: ['Z601', 'Z602', 'Z603', 'Z604'],
```
Comment (Koenkk) [inline devices.js:15495]: Are these 1/2/3/4 gang versions? If yes please add them separately

**Comment by Koenkk** [comment]
Can you provide a link to this device? (link)

Note to self: after this merge: https://github.com/Koenkk/zigbee2mqtt.io/pull/493

File: devices.js (hunk 1 above)
Comment (manhere) [reply devices.js:15495]: no，all these models are the same device

**Comment by manhere** [comment]
> Can you provide a link to this device? (link)
> 
> Note to self: after this merge: [Koenkk/zigbee2mqtt.io#493](https://github.com/Koenkk/zigbee2mqtt.io/pull/493)
https://item.taobao.com/item.htm?id=521229558487
https://n.znds.com/article/6780.html
http://item.gome.com.cn/A0006124103-pop8009585024.html

File: devices.js (hunk 1 above)
Comment (Koenkk) [reply devices.js:15495]: Why is the try/catch here: https://github.com/Koenkk/zigbee-herdsman-converters/pull/1880/files#diff-5520dd49755e0a353d3857b8963a91cb4b8d87a01b6b8eb7bb6cce1c180eac83R15511 ? // dip switch for 1-3 gang -> sounds like there are 1 till 3 gang versions?

Comment (manhere) [reply devices.js:15495]: this switch has a dip-switch under its panel, this dip-switch changes the model and endpoints, so i combined the code

Comment (Koenkk) [reply devices.js:15495]: Thanks

**Comment by Koenkk** [comment]
Thanks!

**Pull request merged**